### PR TITLE
Add section on eunit in erlang doc

### PR DESCRIPTION
**Pull request by edholland**

### CODE REVIEW

File: erlang.html.markdown [hunk 1]
```diff
@@ -299,6 +299,30 @@ CalculateArea ! {circle, 2}. % 12.56000000000000049738
 % The shell is also a process; you can use `self` to get the current pid.
 self(). % <0.41.0>
 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% 5. Testing with EUnit
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% Unit tests can be written using EUnits's test generators and assert macros
```
Comment (vendethiel) [inline erlang.html.markdown]: so, how do you run said tests?

**Comment by edholland** [comment]
updated. Please review again :)

File: erlang.html.markdown [hunk 2]
```diff
@@ -299,6 +299,37 @@ CalculateArea ! {circle, 2}. % 12.56000000000000049738
...
+	?_assertException(error, function_clause, fib(-1)),
+	?_assert(fib(31) =:= 2178309)
+       ].
+
+% EUnit will automatically export to a test() fucntion to allo running the tests
+% in the erlang shell
+fib:test()
+
+% The popular erlang build tool Rebar is also compatible with EUnit
+rebar eunit
```
Comment (vendethiel) [inline erlang.html.markdown]: please comment this, as it's not erlang code

**Comment by edholland** [comment]
updated

**Comment by vendethiel** [comment]
Thanks for copping with my weird requests!

**Comment by vendethiel** [comment]
:purple_heart: :blue_heart: :yellow_heart: :green_heart:

**Pull request merged**